microsoft/atlas-design
# Upgrade to Node 16.6.X

Task: task-[463876](https://dev.azure.com/ceapex/Engineering/_workitems/edit/463876/)

Link: preview-[230](https://design.docs.microsoft.com/pulls/230)

This PR bumps Node version to 16.6.X. Due to the new version's incompatibility with `sass-extract`'s dated dependencies, I have replaced `sass-extract` with `sass-export` to extract the tokens in JSON format. 

Sass-export: https://github.com/plentycode/sass-export

## Testing

1. Visit the [preview link](https://design.docs.microsoft.com/pulls/230). Check out different pages and test all three themes to make sure everything still works as before.
2. Checkout `main` branch, run `npm run build:tokens` to generate `tokens.json`. Save a copy somewhere to compare with the new version. 
3. Checkout this branch `ming/sass-export`. Run `npm i` and  `npm run build:tokens` to generate `tokens.json`. Compare with the version from step 2. You should also be able to run `npm run serve:site` without errors.
4. Open one of the token files and remove the `sass-export-section` comments. Run `npm run build:tokens` and the console should print a warning about `sass-export-section` comments being missing.

Expected result:  The output should be identical except for color values.
